_storage/overview.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: storage
+permalink: /:collection/:path.html
+---
+# A decentralized storage architecture
+
+The Blockstack Network stores application data using a storage service called
```

Comment:
Do we want to call Gaia a "system" rather than a "service" since the latter may connote a centralized server-based resource? 

We seem to use the word "system" in [Gaia's top-level README](https://github.com/blockstack/gaia) with "This document describes the high-level design and implementation of the Gaia storage system".

Reply:
Corrected per group discussion.